src/useObservable.ts

```diff
@@ -1,7 +1,7 @@
 import { observable } from "mobx"
 import { useRef } from "react"
 
-type SupportedValues = object | Map<unknown, unknown> | Array<unknown>
+type SupportedValues = object | Map<any, any> | any[]

```

Comment:
Is it really better in this context to use `any`? If I understand it correctly, the `unknown` is meant for cases like this when you want to ensure some type safety but keeping it up to a consumer.

Reply:
Yea, this is bleeding edge package, no compromises for old stuff :)

Reply:
Should I change it back to unknown then? (although I still see it useful for output types only TBH)

Reply:
Ok, I guess you can keep it like this. I don't see any real harm in it. I have kinda a mindset that `any` is a pure evil and should be avoided at all cost :)